frontend/src/metabase/search/components/SidebarFilter/SidebarFilter.styled.tsx

```diff
@@ -1,37 +1,70 @@
 import styled from "@emotion/styled";
 import { css } from "@emotion/react";
-import { ParameterFieldSet } from "metabase/parameters/components/ParameterWidget/ParameterWidget.styled";
-
-export const DropdownFilterElement = styled(ParameterFieldSet)`
-  height: 40px;
-
-  ${({ theme, fieldHasValueOrFocus }) => {
-    return (
-      fieldHasValueOrFocus &&
-      css`
-        border-color: ${theme.colors.brand[1]};
-        color: ${theme.colors.brand[1]};
-      `
-    );
-  }}
-  &:hover {
-    ${({ theme }) => {
-      return css`
-        background-color: ${theme.colors?.bg[1]};
-        transition: background-color 0.3s;
-        cursor: pointer;
-      `;
-    }}
+import type { HTMLAttributes } from "react";
+import type { ButtonProps } from "metabase/ui";
+import { Button, Stack } from "metabase/ui";
+import FieldSet from "metabase/components/FieldSet";
+import EventSandbox from "metabase/components/EventSandbox";
+
+export const DropdownFieldSet = styled(FieldSet)<{
```

Comment:
`FieldSet` component can't be focused with keyboard (therefore popover can be opened only with mouse).

Reply:
The arrow on the button can be focused and clicked to open the dropdown. I'm using the fieldset to maintain some level of consistency in styling with the legend, but it's definitely due for an upgrade once we get more Mantine components, I've been playing around with the `Menu` component and we can do something with that in the future